concent_api/core/payments/bankster.py

```diff
@@ -254,6 +253,7 @@ def finalize_payment(deposit_claim: DepositClaim) -> Optional[str]:
     return deposit_claim.tx_hash
 
 
+@retry_on_database_error
 def settle_overdue_acceptances(
```

Comment:
Once if there are two requests at the same time. What if it will be more of them? Just curious, in new implementation it will be no up to date any more :)

Reply:
I does not depend on the number of requests.

It's like this:
- N requests come at the same time.
- N processes try to create an object at the same time.
- 1 process succeeds. N-1 processes fail.
- Each of the N-1 processes tries to get the object.
    - Get should succeed because the object now exists. If it fails again, it must be for some other, unknown reason and we don't want to silence it and there's no point in retrying.